security/boot-security/src/main/groovy/yakworks/security/spring/token/TokenController.groovy

```diff
@@ -127,4 +133,11 @@ class TokenController {
             .body(body)
     }
 
+    @ExceptionHandler(Exception.class)
+    def handleException(HttpServletRequest req, HttpServletResponse resp, Exception e) {
+        log.error(e.message)
+        //We dont have access to ProblemHandler here in boot-security. But use a problem to be consistent.
+        Problem problem = new UnexpectedProblem().cause(e).detail(e.message)
```

Comment:
a user not being found is not an UnexpectedProblem, that is a know problem. slow down and get your arms around this so we are not doing so much back and forth here

Reply:
I added this as a general exception handler, will have check for specific UserNotFoundException